File: 2063-vowels-of-all-substrings/2063-vowels-of-all-substrings.java
class Solution {
    public long countVowels(String word) {
        long ans=0;
        int n=word.length();
        for(int i=0;i<n;i++){
            char ch=word.charAt(i);
            if(ch=='a' || ch=='e' || ch=='i' || ch=='o' || ch=='u'){
                ans+=1l*(i+1)*(n-i);
            }
        }
        return ans;
    }
}

File: 2063-vowels-of-all-substrings/2063-vowels-of-all-substrings.cpp
typedef long long ll;
class Solution {
public:
    long long countVowels(string word) {
        ll ans=0;
        int n=word.size();
        for(int i=0;i<n;i++){
            char ch=word[i];
            if(ch=='a' || ch=='e' || ch=='i' || ch=='o' || ch=='u'){
                ans+=1ll*(i+1)*(n-i);
            }
        }
        return ans;
    }
};
